components/Pagination/sub-components/ActionButton.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Colors from '../../Colors';
+
+const StyledButton = styled.a`
+  border: 1px solid transparent;
+  border-radius: 10px;
+  cursor: pointer;
+  font-weight: normal;
+  margin: 0 4px;
+  outline: none;
+  padding: 8px;
+  text-align: center;
+  text-decoration: none;
+  transition: color 0.2s ease-in-out, background-color 0.2s ease-in-out,
+    border 0.2s ease-in-out;
+  user-select: none;
+  white-space: nowrap;
+
+  ${({ 'aria-disabled': ariaDisabled }) =>
```

Comment:
Why not `disabled`?

Reply:
It was disabled in the past, but, the `disabled` prop was only for the styling porpoise, using `aria-disabled` we removed disabled prop since this `aria-disabled` needs to be present when the ActionButton is disabled (for accessibility improvements), so, talking with @ddsilva we decided that using only this prop, even for style or accessibility would be better.

Reply:
I'll not apt for disabling an `ActionButton` only using a disabled prop? Must I use `aria-disabled` instead?

Reply:
Right, since `disabled` attribute do not have any behavior or semantic meaning to an anchor element.
`disabled` attribute only works to form elements.

Reply:
Ok